main.py

```diff
@@ -84,10 +108,88 @@
+                dead_participant_role = discord.utils.find(lambda r: r.id == dead_participant, roles)
+                frozen_participant_role = discord.utils.find(lambda r: r.id == frozen_participant, roles)
+                default_permissions = {
+                    main_guild.default_role: discord.PermissionOverwrite(read_messages=False),
+                    frozen_participant_role: discord.PermissionOverwrite(send_messages=False),
+                    dead_participant_role: discord.PermissionOverwrite(read_messages=True, send_messages=False),
+                    game_master_role: discord.PermissionOverwrite(read_messages=True),
+                    client.user: discord.PermissionOverwrite(read_messages=True,send_messages=True),
+                    **{
+                        member: discord.PermissionOverwrite(read_messages=True) for member in viewers
+                    },
+                }
+
+                # Create a new category if needed
+                if db.get_category() == None:
+                    category = await main_guild.create_category('CC part {}'.format(db.count_categories(), reason='It seems like we couldn\'t use our previous category! Don\'t worry, I just created a new one.')
+                    db.add_category(category)
+                else:
+                    category = db.get_category()
+
+                try:
+                    # Create the text channel
+                    channel = await main_guild.create_text_channel(
+                        name="s{}_".format(config.season) + element.name,
+                        category=category,
+                        overwrites=default_permissions,
+                        reason='CC requested by ' + message.author.mention)
+                    db.add_channel(channel.id,element.owner)
+                    await channel.send(intro_msg)
+
+                    # Set all access rules in the database
+                    for victim in abductees:
+                        db.set_user_in_channel(channel.id,victim,3)
+                    for user in viewers:
+                        if db_get(user,'role') in ['Dead','Spectator']:
+                            db.set_user_in_channel(channel.id,user,4)
+                        elif db_get(user,'frozen') == 1:
+                            db.set_user_in_channel(channel.id,user,2)
+                        else:
+                            db.set_user_in_channel(channel.id,user,1)
+
+                except Exception as e: # Catch any thrown exceptions and send an error to the user.
+                    await message.channel.send('It seems like I\'ve encountered an error! Please let the Game Masters know about this!')
+                    await botspam_channel.send("Oi, Game Masters! Please check the console, I got a problem concerning channel creation for ya to fix.")
+                    raise e # Send the full log to Buddy1913 and his sketchy VM.
```

Comment:
You should probably also send the exception to botspam, or at least without the traceback

Reply:
I don't know how to do that. You'd make me really happy if you could do that for me.
Sounds like a good idea, however!

Reply:
botspam_channel.send(e)

Reply:
Is it THAT simple? Would ALMOST raise the question why you didn't add this yourself just now. 🤔 

Implementing it right now.